bundle-size/test/app.test.js

```diff
@@ -0,0 +1,371 @@
+
+describe('bundle-size', async () => {
+  let probot;
+  let app;
+  const db = dbConnect();
+
+  beforeAll(async () => {
+    await setupDb(db);
+
+    probot = new Probot({});
+    app = probot.load(bundleSizeApp);
+
+    // Return a test token.
+    app.app = () => 'test';
+  });
+
+  beforeEach(async () => {
+    process.env = {
+      MAX_ALLOWED_INCREASE: '0.1',
+    };
+
+    nock('https://api.github.com')
+        .post('/app/installations/123456/access_tokens')
+        .reply(200, {token: 'test'});
+
+    nock('https://api.github.com')
+        .get('/repos/ampproject/amphtml-build-artifacts/contents/bundle-size/' +
+             'OWNERS')
+        .reply(200, require('./fixtures/OWNERS'));
+  });
+
+  afterEach(async () => {
+    await db('checks').truncate();
+  });
+
+  afterAll(async () => {
+    await db.destroy();
+  });
+
+  test('new pull request opened', async () => {
+    const payload = require('./fixtures/pull_request.opened');
+
+    nock('https://api.github.com')
+        .post('/repos/ampproject/amphtml/check-runs', body => {
+          expect(body).toMatchObject({
+            head_sha: '39f787c8132f9ccc956ed465c0af8bc33f641404',
+            name: 'ampproject/bundle-size',
+            output: {
+              summary: 'Calculating new bundle size for this PR…',
+              title: 'Calculating new bundle size for this PR…',
+            },
+          });
+          return true;
+        })
+        .reply(200, {id: 555555});
+
+    await probot.receive({name: 'pull_request', payload});
+
+    expect(await db('checks').select('*')).toMatchObject([
+      {
+        head_sha: '39f787c8132f9ccc956ed465c0af8bc33f641404',
+        base_sha: '263a7fa1188b65d850e6742a63c38a216091d8b2',
+        owner: 'ampproject',
+        repo: 'amphtml',
+        pull_request_id: 19621,
+        installation_id: 123456,
+        check_run_id: 555555,
+        delta: null,
+      },
+    ]);
+  });
+
+  test('approved review submitted by an OWNERS person', async () => {
+    const payload = require('./fixtures/pull_request_review.submitted');
+
+    await db('checks').insert({
+      head_sha: '26ddec3fbbd3c7bd94e05a701c8b8c3ea8826faa',
+      base_sha: '5f27002526a808c5c1ad5d0f1ab1cec471af0a33',
+      owner: 'ampproject',
+      repo: 'amphtml',
+      pull_request_id: 19603,
+      installation_id: 123456,
+      check_run_id: 555555,
+      delta: 0.2,
+    });
+
+    nock('https://api.github.com')
+        .patch('/repos/ampproject/amphtml/check-runs/555555', body => {
+          expect(body).toMatchObject({
+            conclusion: 'success',
+            output: {
+              title: 'Δ +0.20KB | approved by @aghassemi',
+              summary: 'Δ +0.20KB | approved by @aghassemi',
+            },
+          });
+          return true;
+        })
+        .reply(200);
+
+    await probot.receive({name: 'pull_request_review', payload});
+  });
+
+  test('approved review submitted by a non-OWNERS person', async () => {
```

Comment:
a little unclear on what the resulting behavior should be on non-OWNER vs OWNER

Reply:
I changed all the test names to be more descriptive